lnwallet/channel.go

```diff
@@ -1873,7 +1975,12 @@ func (lc *LightningChannel) restoreStateLogs(
 		return err
 	}
 
-	return nil
+	// Restore unsigned acked local log updates so we expect the peer to
+	// sign for them.
+	err = lc.restorePeerLocalUpdates(
```

Comment:
Style nit: can just return the method directly here since it only has a single return value.